### PR TITLE
[PC-1225] - Getting started with MicroPython on the Portenta C33

### PR DESCRIPTION
**Pull request by ArduinoBot**
## What This PR Changes
- This PR adds MicroPython documentation of the Portenta C33 board to Docs. 

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by jcarolinares** [comment]
Hello @jacobhylen just to make sure we are not disrupting the micropython content I would like your approval, thanks :)

File: content/micropython/01.basics/02.board-installation/board-installation.md [hunk 1]
```diff
@@ -277,4 +277,28 @@ Do not disconnect your board during this part of the process, watch the progress
...
+## Portenta C33
+
+To install the MicroPython firmware to the Portenta C33 board, you can use the **Arduino MicroPython Installer**. This tool lets you plug in the board, select it from a menu, and automatically flash the latest firmware or a custom one of your choice to the board. You can download the [Arduino MicroPython Installer here](https://labs.arduino.cc/en/labs/micropython-installer). 
+
+***The Arduino MicroPython Installer is part of [Arduino Labs](https://labs.arduino.cc) and is considered experimental software.***
+
+Before installing the MicroPython firmware, **ensure you have installed the latest Portenta Renesas board package**. To install the board package for the Portenta C33 board, navigate to **Tools > Board > Boards Manager** or click the Boards Manager icon in the left tab of the IDE. In the Boards Manager tab, search for `renesas` and install the latest `Arduino Renesas Portenta Boards` version. 
+
+
+![Installing the Arduino Renesas Portenta Boards core in the Arduino IDE](./assets/portenta-c33-1.png)
```
Comment (jacobhylen) [inline content/micropython/01.basics/02.board-installation/board-installation.md:291]:
```suggestion
![Installing the Arduino Renesas Portenta Board Package in the Arduino IDE](./assets/portenta-c33-1.png)

```

File: content/micropython/01.basics/06.board-examples/board-examples.md [hunk 2]
```diff
@@ -1546,3 +1546,324 @@ If you need to stop the audio streaming, you can call `.stop_streaming()`.
...
+        time.sleep(0.001)
+```
+
+### Real-Time Clock
+
+The `RTC` class in MicroPython provides a way to manage and utilize the Real-Time Clock (RTC) of the Portenta C33 board. This feature is essential for applications that require accurate timekeeping, even when the main processor is not active. The RTC maintains accurate time and date, functioning independently from the main system. It continues to keep track of the time even when the board is powered off, as long as it's connected to a power source like a battery.
+
+#### Initializing the RTC
+
+To use the RTC, create first an RTC object. This object is then used to set or read the current date and time.
```
Comment (jacobhylen) [inline content/micropython/01.basics/06.board-examples/board-examples.md:1822]:
```suggestion
To use the RTC, first create an RTC object. This object is then used to set or read the current date and time.

```

**Review by jacobhylen — APPROVED** [review]
Approving with minor comments, nice work :)

**Comment by jcarolinares** [comment]
Comments addressed,

Merging after Content CI check

**Pull request merged**